config/config.go

```diff
@@ -29,6 +29,11 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+const (
+	// minRangeMaxBytes is the minimum value for range max bytes.
+	minRangeMaxBytes = 1 << 20
```

Comment:
I grabbed the Validate function directly from the [old one](https://github.com/cockroachdb/cockroach/pull/2495/files#diff-ec1cff2fcd2d1176585632d73956682b) which I happened to delete in my previous PR.

I'm not sure we need to lower it here as it's only being used in the cmdline, and stress tests will most likely write the zone configs directly. I'll add a todo next to it in case we ever use it to filter out or modify stored configs.